File: fizzbuzz/src/julia/fizzbuzz.jl
for n in 1:50
    if n % 3 == 0 && n % 5 == 0
        println("fizz buzz")
    elseif n % 3 == 0
        println("fizz")
    elseif n % 5 == 0
        println("buzz")
    else
        println(n)
    end
end

File: fizzbuzz/src/lua/fizzbuzz.lua
for n = 1, 50 do
  if n % 15 == 0 then
    print("fizz buzz")
  elseif n % 3 == 0 then
    print("fizz")
  elseif n % 5 == 0 then
    print("buzz")
  else
    print(n)
  end
end